3p/integration.js

```diff
@@ -165,6 +165,13 @@ const defaultAllowedTypesInCustomFrame = [
   '_ping_',
 ];
 
+// List of ad networks that will manually call `window.context.renderStart` to
+// emit render-start event when ad actually starts rendering. Please add
+// yourself here if you'd like to do so (which we encourage).
+const waitForRenderStart = [
```

Comment:
I like hash better, But I use array because the existing `defaultAllowedTypesInCustomFrame` is an array and I want to keep the style same.